composer.json

```diff
@@ -18,23 +18,28 @@
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "7.3.99"
+            "php": "7.4.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "composer/package-versions-deprecated": true,
+            "infection/extension-installer": true
         }
     },
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0"
-    },
-    "suggest": {
-        "ext-iconv": "*",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "ext-ctype": "*",
         "ext-mbstring": "*"
```

Comment:
Interesting how `ext-mbstring` already was a requirement :D

Reply:
It wasn't… I added it in after you suggested removing the checks and requiring it